package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "file-explorer",
   "main": "./lib/file-explorer",
-  "version": "0.2.3",
```

Comment:
A version number is automatically updated by `apm publish` command. You don't need change this line.
https://atom.io/docs/v0.192.0/hacking-atom-package-word-count